assets/html/evolution/paragraph-gallery.html

```diff
@@ -0,0 +1,333 @@
+    general theory of relativity to model the large-scale structure of the universe.</p>
+  <div class="paragraph-gallery">
+    <div class="gallery-item__wrapper">
+    <span class="paragraph-gallery__item">
+      <img class="paragraph-gallery__image" src="https://placeimg.com/350/350/animals">
+    </span>
+    </div>
+    <div class="gallery-item__wrapper">
+    <span class="paragraph-gallery__item">
+      <img class="paragraph-gallery__image" src="https://placeimg.com/350/350/nature">
+    </span>
+    </div>
+    <div class="gallery-item__wrapper">
+    <span class="paragraph-gallery__item">
+      <img class="paragraph-gallery__image" src="https://placeimg.com/350/350/people">
+    </span>
+    </div>
+    <div class="gallery-item__wrapper">
+    <span class="paragraph-gallery__item">
+      <img class="paragraph-gallery__image" src="https://placeimg.com/350/350/arch">
+    </span>
+    </div>
+  </div>
+  <br><br>
+  <p>He was visiting the United States when Adolf Hitler came to power in 1933 and, being Jewish, did not go back to Germany, where he had been a professor at the Berlin Academy of Sciences. He settled in the United States, becoming an American citizen
+    in 1940. On the eve of World War II, he endorsed a letter to President Franklin D. Roosevelt alerting him to the potential development of "extremely powerful bombs of a new type" and recommending that the U.S. begin similar research. This eventually
+    led to what would become the Manhattan Project. Einstein supported defending the Allied forces, but generally denounced the idea of using the newly discovered nuclear fission as a weapon. Later, with the British philosopher Bertrand Russell, Einstein
+    signed the Russell–Einstein Manifesto, which highlighted the danger of nuclear weapons. Einstein was affiliated with the Institute for Advanced Study in Princeton, New Jersey, until his death in 1955.</p>
+  <div class="paragraph-gallery">
+    <div class="gallery-item__wrapper">
+    <span class="paragraph-gallery__item">
+      <img class="paragraph-gallery__image" src="https://placeimg.com/350/350/animals">
+    </span>
+    </div>
+    <div class="gallery-item__wrapper">
+    <span class="paragraph-gallery__item">
+      <img class="paragraph-gallery__image" src="https://placeimg.com/350/350/nature">
+    </span>
+    </div>
+    <div class="gallery-item__wrapper">
+    <span class="paragraph-gallery__item">
+      <img class="paragraph-gallery__image" src="https://placeimg.com/350/350/people">
+    </span>
+    </div>
+    <div class="gallery-item__wrapper">
+    <span class="paragraph-gallery__item">
+      <img class="paragraph-gallery__image" src="https://placeimg.com/350/350/arch">
+    </span>
+    </div>
+    <div class="gallery-item__wrapper">
+    <span class="paragraph-gallery__item">
+      <img class="paragraph-gallery__image" src="https://placeimg.com/350/350/tech">
+    </span>
+    </div>
+  </div>
+</article>
+
+<div class="evo_c-markup">
```

Comment:
@imanuelgittens 
Remove this code display section `<div class="evo_c-markup">`, we are not using it anymore.